Security/Fundamentals/Security_Principles.mediawiki

```diff
@@ -83,10 +83,11 @@ Changes are detailed in the [https://github.com/mozilla/wikimo_opsec/commits/mas
 * Encrypt data at rest when not actively used by the system.
 * Use full-disk encryption where available.
 * Encrypt credentials storage databases (Ansible Vault, CredStash, etc.)
-* Encrypt traffic in transit (TLS).
+* Encrypt traffic in transit (TLS) from the client to the final destination ([https://en.wikipedia.org/wiki/End-to-end_encryption end-to-end]).
+* Encrypt traffic in transit (TLS) between components of Mozilla's infrastructure when [https://en.wikipedia.org/wiki/End-to-end_encryption end-to-end] encryption is not technically feasible.
```

Comment:
I think this conflicts with the line above. Do you mean "Encrypt traffic between the client and app's network edge if end-to-end is not feasible"? (Allows plain text from LB to app.)